contents/docs/user-guides/experimentation.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Experimentation
+sidebar: Docs
+showTitle: true
+---
+
+<FeatureAvailability availablePlans={['standard', 'enterpriseCloud', 'startup', 'scale', 'enterprise']} />
+
+Experimentation helps you test product changes to your app and find the optimal changes. You can test hypotheses about which product changes will lead to optimal results. You may have heard it before as A/B testing. Examples:
+
+-   You want to test which copy or visual design has the highest conversion.
+-   You want to test which flow leads to the highest number of purchases.
+-   Another use case here?
+
+> 🚧 **Experimentation is currently in beta** and support is limited to our [JavaScript](/docs/integrate/client/js#feature-flags) integration. If you have any feedback, please <a href="https://posthog.com/support" target="_blank">reach out</a>.
+
+## How it works?
+
+With Experimentation, you start with a hypothesis on how to improve your product (e.g. changing the flow, changing copy, changing visual styles, ...) and a goal (e.g. improve conversion, improve sign ups, ...).
+
+-   Once you have those, you can create an Experiment which will have a control group and any number of experimentation variants (at least one).
+-   You can select which users will participate in the Experiment (e.g. paid users, users in a given country, or all users).
+-   Based on your user selection and your goal, we will estimate the number of users exposed and the number of days required for your Experiment to reach significant results.
+-   You can tweak your settings to speed up the experiment or adjust user exposure (e.g. you may not want certain users exposed to your experiment).
+-   The actual experiment and what your user sees is controlled in your own code (see below on [Creating an experiment](#creating-an-experiment)). You ask PostHog which group does this user belong to and then you show any relevant changes in your app or website. Because this is done in your code you can test any number of things, from changing copy in a button, to taking the user to a different page, or showing them a completely different experience.
```

Comment:
```suggestion

The actual experiment and what your user sees is controlled in your own code (see below on [Creating an experiment](#creating-an-experiment)). You ask PostHog which group does this user belong to and then you show any relevant changes in your app or website. Because this is done in your code you can test any number of things, from changing copy in a button, to taking the user to a different page, or showing them a completely different experience.
```